doc/_static/theme_override.css

```diff
@@ -20,3 +20,12 @@ div[class^="highlight"] a:hover {
 .rst-versions.shift-up {
     overflow-y: visible;
 }
+
+a[class^="sphx-glr-backref-module-"] {
+    text-decoration: none;
+    background-color: rgba(0, 0, 0, 0) !important;
```

Comment:
why `!important` here? It makes it hard to override by package developers, I think (I'm no CSS expert so please bear with me :-))

Reply:
This is just for building the SG docs, it shouldn't affect actual use of SG by downstream libraries. The one CSS change that will is in gallery.css